tests/test_renderers.py

```diff
@@ -0,0 +1,116 @@
+import json
+
+import django
+# We run django.setup() in order to auto populate the base django's app models for testing purposes
+import pytest
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework_json_api import serializers
+
+from tests.django_mocking.models import BasicModel
+
+try:
+    django.setup()
```

Comment:
I might be missing something - but I think @jamesfneyer was able to get the 'test' django project loaded in the test_filters.py tests without explicitly calling django.setup(). Might be because he used a django TestCase instead of plain pytest? Would be nice to get rid of this boilerplate, I see it in a few other unit tests files.

Reply:
test_filters.py was able to avoid the boilerplate because it is not utilizing any pytest fixtures.  Once you start to use pytest fixtures in a subclass of a TestCase, things go downhill.  There appear to be workarounds, but they look to be more boilerplate than the `django.setup()` we have here.

`REF: https://github.com/pytest-dev/pytest-django/issues/570#issuecomment-360933775`